universal-login-ops/src/cli/cli.js

```diff
@@ -23,6 +36,11 @@ const commandLineBuilder = yargs
     (argv) => {
       connectAndExecute(argv.nodeUrl, argv.privateKey, deployToken).catch(console.error);
     })
+  .command('send', 'Sends funds to specified address',
+    () => {},
```

Comment:
yargs
      .positional('to', {
        describe: '...'
      }).positional('amount', {
        describe: '...'
      }).positional('currency', {
        describe: '...',
        default: ...
      })